source/examples/uiconcept/main.cpp

```diff
@@ -39,6 +40,14 @@ int main(int argc, char *argv[])
     // Load and show QML
     engine.load(QUrl::fromLocalFile(engine.qmlToolboxModulePath() + "/examples/uiconcept/window.qml"));
     
+    auto windows = app.allWindows();
+	if (!windows.empty())
+	{
+		auto window = windows.first();
```

Comment:
I don't think so. But why not just iterate over all windows?
Also I would prefer to use the terms "fullscreen mode" and "windowed mode" instead of "normal"